addons/account/models/res_config_settings.py

```diff
@@ -147,8 +147,6 @@ class ResConfigSettings(models.TransientModel):
 
     def set_values(self):
         super(ResConfigSettings, self).set_values()
-        if self.group_multi_currency:
-            self.env.ref('base.group_user').write({'implied_ids': [(4, self.env.ref('product.group_sale_pricelist').id)]})
```

Comment:
Advanced price lists are not necessary - basic price list is activated when group_multi_currency is activated... see `res_currency.write()`

Reply:
see `_toggle_group_multi_currency`